monero/transaction/__init__.py

```diff
@@ -199,9 +210,37 @@ def _scan_pubkeys(svk, psk, stealth_address, amount, encamount):
                     *map(operator.methodcaller("addresses"), wallet.accounts)
                 )
             )
+        '''
+        pre hard fork:
+        { 
+          "target": {
+            "key": "ea3f..."
+          }
+        }
+        post hard fork:
+        { 
+          "target": {
+            "tagged_key": {
+              "key": "ea3f...",
+              "view_tag": "a1"
+            }
+          }
+        }
+        '''
         outs = []
         for idx, vout in enumerate(self.json["vout"]):
-            stealth_address = binascii.unhexlify(vout["target"]["key"])
+            #see if post hard fork json structure present:
+            try:
+                if vout["target"]["tagged_key"]:
+                    #post fork transaction
+                    stealth_address = binascii.unhexlify(vout["target"]["tagged_key"]["key"])
+                    on_chain_vt = binascii.unhexlify(vout["target"]["tagged_key"]["view_tag"])
```

Comment:
probably need to keep this `on_chain_vt` a hex string for the equality check against `vt` downstream to work :)

Or alternatively, can keep this as is, and then instead of `vt = binascii.hexlify(vt_full)[0:2]` later on, grab just the first byte of `vt_full` in binary.. but I'm not sure how to do that in this code. That's probably the ideal approach